source/slang/slang-hash-utils.h

```diff
@@ -4,19 +4,22 @@
 #include "../core/slang-basic.h"
 #include "../core/slang-md5.h"
 #include "../core/slang-digest.h"
+#include "../core/slang-char-util.h"
 
 namespace Slang
 {
+    using slang::Digest;
+
     // Compute the hash for an UnownedStringSlice
-    inline slang::Digest computeHashForStringSlice(UnownedStringSlice text)
+    inline Digest computeHashForStringSlice(UnownedStringSlice text)
```

Comment:
`computeDigestForStringSlice`

Reply:
I think we should just have a `DigestUtil` class here and move all these functions into it as static members. That way we can clean the naming up a lot, similar to what `StringUtil` does.
```
class DigestUtil
{
     static Digest fromString(UnownedStringSlice str);
     static String toString(const Digest& digest);
     static Digest combine(const Digest& d0, const Digest& d1);
     static Digest computeDigestForString(UnowendStringSlice text);
}

```